apps/web/scripts/generate-cms-dictionaries.ts

```
@@ -0,0 +1,65 @@
+}) {
+  try {
+    const dirPath = `./dictionaries/${lang}/blog/${category}`;
+    const fileName = `${slug}.json`;
+    const filePath = path.resolve(dirPath, fileName);
+    const canonicalUrl = `https://bitlauncher.ai/${lang}/${category}/${slug}`;
+    const shortLink = (await generateShortLink(canonicalUrl, false)).data
+      ?.shortLink;
+
+    const data = await fs.promises.readFile(filePath, "utf-8");
+    const json = JSON.parse(data);
+    json["shortLink"] = shortLink || "";
```

Biome

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)